### ISSUE TITLE
Stats: static front page stats are labeled with the page title instead of "Home page"

### ISSUE THREAD

**Issue by designsimply** [Automattic/jetpack#41509]
Steps to reproduce:
1. Set a static front page and a posts page
2. Log out and load each page a few times to register some views for them<br><sup>(fewer for one of them so you can tell them apart easily when you check stats for them next)</sup>
3. Log in and go to https://wordpress.com/stats/day for the site
4. Look in the "Posts & Pages" box and check to see how the stats are labeled

<strong>Result:</strong> stats for the static front page are filed under that page's title even though it is the home page and stats for the posts page are labeled "Home page / Archives" even though it is not currently the home page.

![screen shot 2015-08-27 at 6 12 27 pm](https://cloud.githubusercontent.com/assets/8658164/9527454/35dbbe24-4ce7-11e5-91f8-392e518ddd1a.png)

Also reported at https://en.forums.wordpress.com/topic/why-does-home-pagearchives-show-up-in-my-stats?replies=11

<blockquote>We should adjust the label for "Home Page / Archive" to just read "Archive" in these cases.</blockquote>

<blockquote>Preferably, views on my home page would be counted toward "Home Page" (rather than the static page title). It's easy to forget that page's title and wonder what that page is (especially if it hasn't been given an obvious title like "Home").

If a site has a Posts page set, views on the posts page could be called "Posts Page / Archive" or "Blog Page / Archive" and link to that posts page. If there isn't a Posts page for the site, it could be "Archive" and not link anywhere.</blockquote>

/hat tip samjhamdy and rachelmcr for the original report and testing

**Comment by supernovia** [comment]
Thanks @gibrown ; do I need to move this back to wp-calypso? I'm just catching up on pings here and noticed another stats issue I'd moved to Jetpack got moved back.

**Comment by jeherve** [comment]
Yup, since the Stats screen is developed in the Calypso repo, it's probably better there.
